saleor/graphql/checkout/subscriptions.py

```diff
@@ -0,0 +1,69 @@
+from django.conf import settings
+
```

Comment:
What about tests related to this subscription? Should we have it here? Like the test for building the subscription payload?

Reply:
Which tests exactly?

Reply:
I'll move these tests then.